core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -78,14 +78,13 @@ if [ -z "$JBOSS_USER" ]; then
 fi
 
 # Check wildfly user
-id $JBOSS_USER > /dev/null 2>&1
-if [ $? -ne 0 -o -z "$JBOSS_USER" ]; then
+if ! id $JBOSS_USER > /dev/null 2>&1 || [ -z "$JBOSS_USER" ]; then
```

Comment:
If you look at: https://github.com/wildfly/wildfly-core/pull/4647/files#diff-2365dc492fe86de1f4e9838f06559da1b1ef830bf6f3955f7b66484b90c7a6e5R73
the change is different although I guess the same tooling has been used to suggest changes. Correct?

Reply:
Yes, it was giving a warning on this line.

Reply:
I think I made mistake, I need to change the same line there as well. Thanks for noticing it.

Reply:
Changes done.